libcli.spec

```diff
@@ -1,7 +1,7 @@
 Version: 1.10.1
 Summary: Cisco-like telnet command-line library
 Name: libcli
-Release: 1
+Release: 2
```

Comment:
As a bug fix I think this should be 1.10.2, rather than release-2. The Release number is an RPM spec feature, not a version number.
From https://rpm-packaging-guide.github.io/: "The number of times this version of the software was released. Normally, set the initial value to 1%{?dist}, and increment it with each new release of the package. Reset to 1 when a new Version of the software is built."